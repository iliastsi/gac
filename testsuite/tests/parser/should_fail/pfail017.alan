hello() : proc
  foo  : int[100];
  foo2 : byte
{
}
