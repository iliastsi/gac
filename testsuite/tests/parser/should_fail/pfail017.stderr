pfail017.alan:4:1: Error: Parse error on input `{'
	Probably missing `;'

